guidelines/sample-server/files/models.json

```diff
@@ -81,5 +81,23 @@
         "format": "double"
       }
     }
+  },
+  "ErrorMessage": {
+    "description": "A shared model for error message",
+    "required" : ["error-code","reason"],
+    "properties": {
+      "error-code": {
```

Comment:
Which languages support dashes in variable names? The common practice is to use camel case or snake case.